### PR TITLE
Add PlasmaPy to members list

### PR DESCRIPTION
**Pull request by SolarDrew**
PR to have PlasmaPy join OpenAstronomy.

@namurphy, @tulasinandan, @StanczakDominik, etc, corrections/improvements welcome.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -64,6 +64,10 @@ poliastro_url: http://poliastro.readthedocs.io/
 poliastro_mailinglist_dev: https://groups.io/g/poliastro-dev
 poliastro_riot: https://riot.im/app/#/room/#poliastro:matrix.org
 
+plasmapy_url: http://plasmapy.org
+plasmapy_mailinglist: http://groups.google.com/group/plasmapy
+plasmapy_riot: https://riot.im/app/#/room/#pyastro:matrix.org
```
Comment (Cadair) [inline _config.yml]: this is wrong!

Comment (SolarDrew) [reply _config.yml]: Damnit! Too many different things going on with too many similar names.

**Comment by SolarDrew** [comment]
Good call. I am bad at catchy names late at night.

**Comment by tulasinandan** [comment]
I like this suggestion as well.

**Pull request merged**